python/starter_kit/README.md

```diff
@@ -8,3 +8,17 @@ Dans cette fiche, nous allons voir quelles sont les étapes pour bien démarrer
 ## Table des matières
 
 - [Comment bien commencer avec python](fr/START_WELL.md)
+
+
+# Starting well with Python
+
+> Originally written by [TNtube](https://github.com/TNtube)
```

Comment:
Following @Tran-Antoine mind, I think this formulation could be better to be coherent (https://github.com/readthedocs-fr/notions/tree/master/poo/abstraction)
```suggestion
> Original article authored by [TNtube](https://github.com/TNtube)
```